compiler/testData/diagnostics/tests/nullabilityAndSmartCasts/InfixCallNullability.fir.kt

```diff
@@ -11,31 +11,31 @@ operator infix fun A?.times(i : Int) {}
 fun test(x : Int?, a : A?) {
     x.<!NONE_APPLICABLE!>plus<!>(1)
     x?.plus(1)
-    x <!NONE_APPLICABLE!>+<!> 1
-    <!UNSAFE_CALL!>-<!>x
+    <!NONE_APPLICABLE!>x + 1<!>
+    <!UNSAFE_CALL!>-x<!>
     x<!UNSAFE_CALL!>.<!>unaryMinus()
     x?.unaryMinus()
 
     a<!UNSAFE_CALL!>.<!>plus(1)
     a?.plus(1)
-    a <!UNSAFE_INFIX_CALL!>plus<!> 1
-    a <!UNSAFE_OPERATOR_CALL!>+<!> 1
-    <!UNSAFE_CALL!>-<!>a
+    <!UNSAFE_CALL!>a plus 1<!>
+    <!UNSAFE_CALL!>a + 1<!>
+    <!UNSAFE_CALL!>-a<!>
     a<!UNSAFE_CALL!>.<!>unaryMinus()
     a?.unaryMinus()
 
     a<!UNSAFE_CALL!>.<!>div(1)
-    a <!UNSAFE_OPERATOR_CALL!>/<!> 1
-    a <!UNSAFE_INFIX_CALL!>div<!> 1
+    <!UNSAFE_CALL!>a / 1<!>
+    <!UNSAFE_CALL!>a div 1<!>
     a?.div(1)
 
     a.times(1)
     a * 1
     a times 1
     a?.times(1)
 
-    1 <!UNSAFE_OPERATOR_CALL!>in<!> a
-    a <!UNSAFE_INFIX_CALL!>contains<!> 1
+    <!UNSAFE_CALL!>1 in a<!>
+    <!UNSAFE_CALL!>a contains 1<!>
```

Comment:
All changes in this file have different diagnostics, which look not good to me.

Reply:
Thanks! With the latest change this file has been reverted.